draft-ietf-oauth-device-flow.xml

```diff
@@ -115,20 +116,40 @@
     <section anchor="Introduction" title="Introduction">
 
       <t>
-	The device flow is suitable for clients executing on devices that
-	do not have an easy data-entry method and where the client is 
-	incapable of receiving incoming requests from
-	the authorization server (incapable of acting as an HTTP server).
-      </t> 
+        This OAuth 2.0 protocol flow for browserless and input constrained
+        devices, often referred to as the device flow, enables OAuth clients to
+        request user authorization from devices that have an internet
+        connection, but don't have an easy input method (such as a smart TV,
+        media console, picture frame, or printer), or lack a suitable browser
+        for a more traditional OAuth flow. This authorization flow instructs the
+        user to action the authorization request on a secondary device, such as
+        a smartphone.
+      </t>
 
       <t>
-	Instead of interacting with the end-user's user-agent, the client
-	instructs the end-user to use another computer or device and connect
-	to the authorization server to approve the access request.  Since the
-	client cannot receive incoming requests, it polls the authorization
-      server repeatedly until the end-user completes the approval process.</t> 
+        The device flow is not intended to replace browser-based OAuth in native
```

Comment:
In the following sentence there is a glitch: 

"
The device flow is not intended to replace browser-based OAuth in native apps on capable devices (like smartphones), those apps should follow the practices specified in OAuth 2.0 for Native Apps OAuth 2.0 for Native Apps [I-D.ietf-oauth-native-apps].
"

It should read as: 

"
The device flow is not intended to replace browser-based OAuth in native apps on capable devices (like smartphones). Those apps should follow the practices specified in OAuth 2.0 for Native Apps [I-D.ietf-oauth-native-apps].
"